mvrx/src/main/kotlin/com/airbnb/mvrx/MvRxLifecycleAwareFlow.kt

```diff
@@ -0,0 +1,33 @@
+@file:Suppress("EXPERIMENTAL_API_USAGE")
+
+package com.airbnb.mvrx
+
+import androidx.lifecycle.Lifecycle
+import androidx.lifecycle.LifecycleObserver
+import androidx.lifecycle.LifecycleOwner
+import androidx.lifecycle.OnLifecycleEvent
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.combineTransform
+
+/**
+ * Emits values from the source flow only when the owner is started.
+ * When the owner transitions to started, the most recent value will be emitted.
+ */
+fun <T : Any> Flow<T>.flowWhenStarted(owner: LifecycleOwner): Flow<T> {
+    val startedFlow = MutableStateFlow(false)
```

Comment:
@denis-bezrukov I didn't do this because addObserver should bring it up to date according to the docs
https://developer.android.com/reference/kotlin/androidx/lifecycle/Lifecycle#addobserver

Reply:
@gpeal Why not using [`owner.whenStarted()`](https://developer.android.com/reference/kotlin/androidx/lifecycle/package-summary#whenstarted_1)?

Reply:
@jpventura Because that launches a new coroutine every time it is started.